hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/SCMPipelineMetrics.java

```diff
@@ -137,11 +150,16 @@ void incNumPipelineCreated() {
   }
 
   /**
-   * Get the number of pipeline created.
-   * @return number of pipeline
+   * Increments the number of bytes by delta that write into the pipeline.
    */
-  long getNumPipelineCreated() {
-    return numPipelineCreated.value();
+  void incNumPipelineBytesWrite(Pipeline pipeline, long delta) {
```

Comment:
The reported BytesWritten is delta or sum up?

Reply:
I have updated the logic here.